### PR TITLE
Ensure spectator camera preview is up when reopening tablet

### PR DESCRIPTION
**Pull request by davidkelly**
If you dismissed the tablet with the spectator camera running, then opened it again, the preview was not being shown.  Now it is.  While there, also made sure we stop updating the preview window when it isn't visible, as that is just a waste of resources.

### CODE REVIEW

**Comment by davidkelly** [comment]
Test Plan

In HMD, bring up spectator camera
start the camera and then close tablet
spectator camera should of course still be running
reopen tablet and spectator camera
the preview window should be updating, with the appropriate image (hmd or camera view)
do this a couple times, making sure to try with the Monitor Shows switch in both positions

File: interface/src/ui/ResourceImageItem.cpp [hunk 1]
```diff
@@ -66,15 +66,20 @@ void ResourceImageItemRenderer::synchronize(QQuickFramebufferObject* item) {
         _ready = resourceImageItem->getReady();
         readyChanged = true;
     }
-
+    bool visibleChanged = false;
+    if (_visible != resourceImageItem->isVisible()) {
+        _visible = resourceImageItem->isVisible();
+        visibleChanged=true;
```
Comment (zfox23) [inline interface/src/ui/ResourceImageItem.cpp]: Spacing around equals

Comment (davidkelly) [inline interface/src/ui/ResourceImageItem.cpp]: doh! thx!

**Comment by davidkelly** [comment]
merging as this will be QA'd in the big branch PR

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/1509/HighFidelity-Beta-PR10932-fcf4bfd59a65c2fe5342681bd79e30b41b54c208.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/1509/HighFidelity-Beta-PR10932-fcf4bfd59a65c2fe5342681bd79e30b41b54c208.exe

**Pull request merged**